testbook/execute.py

```diff
@@ -6,15 +6,42 @@
 from testbook.client import TestbookNotebookClient
 
 
-@pytest.fixture
-def notebook_loader():
-    @contextmanager
-    def notebook_helper(nb_path, **kwargs):
-        with open(nb_path) as f:
+class notebook_loader:
```

Comment:
I like how you made this class both a decorator and a context manager.

Reply:
Thanks! It was inspired by `mock.patch` which can also be used as a decorator and contextmanager.

https://github.com/python/cpython/blob/493bf1cc316b0b5bd90779ecd1132878c881669e/Lib/unittest/mock.py#L1244